packages/polaris-viz/src/components/shared/HorizontalStackedBars/HorizontalStackedBars.tsx

```diff
@@ -147,6 +185,17 @@ export function HorizontalStackedBars({
           </Fragment>
         );
       })}
+      {!isSimple && (
```

Comment:
Just spit balling here, I wonder if we still want a way to show/hide labels even for non-simple charts.

What do you think?

Reply:
@envex I added an option to the `Theme`  that supports hiding group labels.